test/css.js

```diff
@@ -41,3 +41,95 @@ QUnit[ ( jQueryVersionSince( "3.4.0" ) && typeof Proxy !== "undefined" ) ? "test
 
 	delete jQuery.cssProps.devoHat;
 } );
+
+QUnit.test( "jQuery.css with numbers", function( assert ) {
+	assert.expect( 5 );
+
+	var whitelist = [
+		"margin",
+		"marginTop",
+		"marginRight",
+		"marginBottom",
+		"marginLeft",
+		"padding",
+		"paddingTop",
+		"paddingRight",
+		"paddingBottom",
+		"paddingLeft",
+		"top",
+		"right",
+		"bottom",
+		"left",
+		"width",
+		"height",
+		"minWidth",
+		"minHeight",
+		"maxWidth",
+		"maxHeight",
+		"border",
+		"borderWidth",
+		"borderTop",
+		"borderTopWidth",
+		"borderRight",
+		"borderRightWidth",
+		"borderBottom",
+		"borderBottomWidth",
+		"borderLeft",
+		"borderLeftWidth"
```

Comment:
These are all the possible strings that report `true` when passed to `isAutoPx()`.